packages/pn-validator/src/__test__/ValidatorBuilder.test.ts

```diff
@@ -101,4 +103,28 @@ describe('Test ValidatorBuilder', () => {
     const results = dummyValidatorBuilder.validate('no match', {});
     expect(results).toBe('Custom error message');
   });
+
+  it('isRequired - check if validate works (value not required)', () => {
```

Comment:
You can remove these three tests. The required rule is tested into the RequireRule test and the custom message is already tested above